lib/pages/forward/forward_item.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/pages/chat_list/chat_list_item_style.dart';
+import 'package:fluffychat/pages/forward/forward_item_style.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/matrix_locals.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:linagora_design_flutter/colors/linagora_sys_colors.dart';
+import 'package:matrix/matrix.dart';
+
+class ForwardItem extends StatelessWidget {
+  final Room room;
+  final bool selected;
+  final void Function()? onTap;
+
+  const ForwardItem(
+    this.room,
+    {
+      this.selected = false,
+      this.onTap,
+      Key? key,
+    }
+  ) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final unread = room.isUnread || room.membership == Membership.invite;
+    final displayName = room.getLocalizedDisplayname(
+      MatrixLocals(L10n.of(context)!),
+    );
+    return Material(
+      borderRadius: BorderRadius.circular(AppConfig.borderRadius),
+      clipBehavior: Clip.hardEdge,
+      color: Colors.transparent,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 8),
+        child: Theme(
+          data: ThemeData(
+            splashColor: Colors.transparent,
+            highlightColor: Colors.transparent,
+          ),
+          child: ListTile(
+            contentPadding: EdgeInsets.zero,
+            title: Row(
+              children: [
+                SizedBox(
+                  width: ForwardItemStyle.avatarSize,
+                  child: Stack(
+                    children: [
+                      Avatar(
+                        mxContent: room.avatar,
+                        name: displayName,
+                      ),
```

Comment:
For this, we can make sure in the case avatar null, we will use our `text` avatar? @sherlockvn @nqhhdev

Reply:
yes